ccd-definition/AuthorisationCaseField/CareSupervision/solicitor.json

```diff
@@ -1689,5 +1689,33 @@
       "[CHILDSOLICITORM]", "[CHILDSOLICITORN]", "[CHILDSOLICITORO]"
     ],
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "refusedOrdersTabHeading",
```

Comment:
Maybe EPSMANAGING rather than respondents + childsolicitors

Reply:
Moved this to EPSMANAGIN